packages/redux/src/users/returns/types/actions.types.ts

```diff
@@ -0,0 +1,26 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type { BlackoutError } from '@farfetch/blackout-client';
+import type { UserReturnsResultNormalized } from '../../../entities';
+
+//
+// Fetch all user returns
```

Comment:
This description can be removed. We usually have one description only for the type that contains all possible actions (in this case, `FetchUserReturnsAction`) and the description is something like this: `
/**
 * Actions dispatched when the fetch user returns request is made.
 */`